### PR TITLE
Import from Rhythmbox playlist fix

### PR DESCRIPTION
**Pull request by daschuer**
This fixes a regression since https://github.com/mixxxdj/mixxx/pull/11230 and Mixxx 2.3.4
that bulk importing tracks from playlists was broken.

### CODE REVIEW

File: src/library/rhythmbox/rhythmboxfeature.cpp [hunk 1]
```diff
@@ -91,13 +91,14 @@ RhythmboxFeature::~RhythmboxFeature() {
...
 
-BaseSqlTableModel* RhythmboxFeature::getPlaylistModelForPlaylist(const QString& playlist) {
-    BaseExternalPlaylistModel* pModel = new BaseExternalPlaylistModel(
-                                            this, m_pLibrary->trackCollections(),
-                                            "mixxx.db.model.rhythmbox_playlist",
-                                            "rhythmbox_playlists",
-                                            "rhythmbox_playlist_tracks",
-                                            m_trackSource);
+std::unique_ptr<BaseSqlTableModel>
+RhythmboxFeature::createPlaylistModelForPlaylist(const QString& playlist) {
```
Comment (Swiftb0y) [inline src/library/rhythmbox/rhythmboxfeature.cpp:95]: just to make sure I understand the fix. The issue was basically that an oversight during refactoring resulted in the wrong member function being called?

**Review by Swiftb0y — COMMENTED** [review]
can you spend some more time making clazy happy? I think you'll need to add the appropriate `override`s to all member functions of the class.

File: src/library/rhythmbox/rhythmboxfeature.cpp (hunk 1 above)
Comment (daschuer) [reply src/library/rhythmbox/rhythmboxfeature.cpp:95]: Yes, the getPlaylistModelForPlaylist() is no longer called, instead the default base class implementation is used returning null.

**Review by Swiftb0y — APPROVED** [review]
code LGTM. I don't use rythmbox so I can't really verify this fix easily

**Comment by daschuer** [comment]
I can. And the coding mistake was obvious.
Merge?

**Comment by Swiftb0y** [comment]
Sure...

**Pull request merged**